.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
```

Comment:
this might not be a bad idea to run on every push, i.e. per commit on a PR

does this run just once on a PR right now?

Reply:
yep it appears to be running on every commit on a PR @atvaccaro